Add Wynnbuilder IDs

ParkerHitch opened the pull request:
Added Wynnbuilder's IDs to items for exporting builds.

ParkerHitch committed:
Added Wynnbuilder IDs

Added Wynnbuilder IDs to items for exporting builds.

ParkerHitch committed:
Improved Wynnbuilder IDs

Made the one line of code that assigns the IDs better.

P0keDev commented on src/main/kotlin/com/wynntils/athena/routes/managers/ItemManager.kt:
```diff
@@ -1,12 +1,16 @@
 object ItemManager {
 
     fun convertItem(input: JSONObject): JSONOrderedObject {
+        val wynnbuilderIDs = FileReader(javaClass.getResource("wynnbuilderIDs.json").file).readText().asJSON<JSONObject>()
```
It would be better to load this file once as opposed to every time convertItem is called

ParkerHitch replied:
Oh yeah good catch. I will change that.

ParkerHitch committed:
Changed to Only Open Json Once

Changed to only open the wynnbuilderIDs json once instead of every time convertItem was called.

HeyZeer0 commented:
Doesn't make sense for this to be on Athena's end, WynnBuilder should be the one to translate the original names to their id format. Along with that, I don't want to keep updating this list at all.
